### ISSUE TITLE
Can you provide the pretrained model?

### ISSUE THREAD

**Issue by Angknpng** [ZZ2490/SPNet#2]
excellent work! However, the pretrained model of SPNet is missing.

**Comment by 2490o** [comment]
Of course, we will upload all the code and weight files before the end of the month at the latest. Thank you for your attention.

发件人：wkkppp ***@***.***>
发送日期：2023-11-12 19:30:01
收件人：ZZ2490/SPNet ***@***.***>
抄送人：Subscribed ***@***.***>
主题：[ZZ2490/SPNet] Can you provide the pretrained model? (Issue #2)

excellent work! However, the pretrained model of SPNet is missing.

**Comment by 55-svg** [comment]
Do you know where the evaluation dataset is？

**Comment by Angknpng** [comment]
> Do you know where the evaluation dataset is？

I think you can find the corresponding evaluation dataset at the link below.
https://github.com/jiwei0921/RGBD-SOD-datasets

**Comment by 55-svg** [comment]
Thank you, but I only saw the training dataset and the test dataset, the evaluation dataset is needed in the training process. Do you know how to get the evaluation data set? In addition, do you have the download address of pretrained model?

**Comment by Angknpng** [comment]
Since I am not the author of this paper, you would be better to contact the author for specific details. Also, in my experience, the evaluation data set may be the test data set when training.





------------------ Original ------------------
From: 55-svg ***@***.***&gt;
Date: Thu,May 16,2024 5:41 PM
To: ZZ2490/SPNet ***@***.***&gt;
Cc: Angknpng ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [ZZ2490/SPNet] Can you provide the pretrained model? (Issue #2)





 
Thank you, but I only saw the training dataset and the test dataset, the evaluation dataset is needed in the training process. Do you know how to get the evaluation data set? In addition, do you have the download address of pretrained model?

**Comment by 55-svg** [comment]
What do you mean by test datasets during training? There is no one in this direction around me. Are you from China? Can you add a contact information? Thank you very much!
